### ISSUE TITLE
Gencode version for extended annotation

### ISSUE THREAD

**Issue by mschilli87** [guigolab/gencode-cls-master-table#1]
The `README` claims

> The enhanced annotation has been obtained by adding to the latest Gencode annotation 

but the linked file refences Gencode v. 43 which is 2.5 years old. The current version is v. 45, which is almost a year old already as well. A quick check revealed only 17 new transcripts between these two releases, but I still found the statement confusing.

Do you plan to publish the code used to extend the gencode annotation so that users could merge your  novel transcripts into any newer/future Genocode releases?

**Comment by TamaraPerteghella** [comment]
Dear [mschilli87](https://github.com/mschilli87),
Sorry for the delayed answer!

Gencode v43 was indeed the reference when we started with the analysis. On its own, Gencode v43 already integrates many of the transcripts we generated (as v27 was the reference geneset at the time the panel was designed and manual integration is a continuous and laborious process than never stopped since). Thanking you for pointing this inconsistency, we changed the sentence in order to make it coherent with the actual state of things. 

Finally, yes, we will eventually make all codes available!
